### PR TITLE
Update rpm-development-guide.md

### PR DESCRIPTION
**Pull request by ddavila0**
In the Git repos section I've changed the syntax to reference git repos, specifically I changed the option 'url' for 'repo' and added the 'tarball' option

### CODE REVIEW

**Comment by matyasselmeci** [comment]
We already have a section on github repos: https://opensciencegrid.org/technology/software/rpm-development-guide/#github-repos

Since we use github repos more often than other git repos, I'd be open to switching the order of those two sections...

**Comment by ddavila0** [comment]
Oh! sorry I didn't see the github section. Changing the order seems like a good idea

Diego

**Comment by brianhlin** [comment]
@ddavila0 could you revert your changes and swap the sections around as suggested?

**Comment by brianhlin** [comment]
@ddavila0 ping, see my comment above

**Comment by ddavila0** [comment]
Sorry I missed your previous comment, I've pushed the changes now

File: docs/software/rpm-development-guide.md [hunk 1]
```diff
@@ -179,83 +179,84 @@ $ sha1sum /p/vdt/public/html/upstream/<PACKAGE>/<VERSION>/<FILE>
...
-###### GitHub repos
+###### Git repos
 
 !!! warning
     OSG software policy requires that all Git and GitHub repos used for building software have mirrors at the UW.
     Many software repos under the [opensciencegrid GitHub organization](https://github.com/opensciencegrid) are already mirrored.
     If you are uncertain, or have a new project that you want mirrored, send email to <osg-software@opensciencegrid.org>.
 
 !!! note
+    You can use a shorter syntax for GitHub repos -- see below.
```
Comment (matyasselmeci) [inline docs/software/rpm-development-guide.md]:
```suggestion
    You can use a shorter syntax for GitHub repos -- see above.
```

**Comment by ddavila0** [comment]
great catch, thank you @matyasselmeci

**Pull request merged**